### DISCUSSION TITLE
Enabling SpamAssassin after initial setup

### DISCUSSION THREAD

**Discussion by gdm85** [docker-mailserver/docker-mailserver#3300] [Q&A] [answered]
Hi there!

After editing `mailserver.env` with `ENABLE_SPAMASSASSIN=1` and restarting docker compose, is there anything else needed to enable SpamAssassin?

In general I was looking for a way to verify that SpamAssassin is running and working.

Thanks!

**Comment by casperklein** [comment, marked as answer]
SA should work out of the box when enabled.

> In general I was looking for a way to verify that SpamAssassin is running and working.

Send yourself a [SPAM](https://en.wikipedia.org/wiki/GTUBE) mail and see if it's flagged.

**Reply by gdm85** [reply]
Thanks; will that affect the score related to my sender address? I assume not.

**Reply by casperklein** [reply]
I don't think so. The GTUBE signature is explicitly for testing.
